src/components/application_manager/src/commands/mobile/set_global_properties_request.cc

```diff
@@ -276,6 +276,30 @@ void SetGlobalPropertiesRequest::on_event(const event_engine::Event& event) {
   }
 }
 
+bool SetGlobalPropertiesRequest::PrepareResultForMobileResponse(
+    ResponseInfo& ui_info, ResponseInfo& tts_info) const {
+  const bool result =
+      CommandRequestImpl::PrepareResultForMobileResponse(ui_info, tts_info);
+  if (ui_info.is_ok && tts_info.is_unsupported_resource) {
+    return true;
+  }
+  return result;
+}
+
+mobile_apis::Result::eType
+SetGlobalPropertiesRequest::PrepareResultCodeForResponse(
+    const ResponseInfo& ui_info, const ResponseInfo& tts_info) {
+  if (ui_info.is_ok && tts_info.is_unsupported_resource) {
+    return mobile_apis::Result::WARNINGS;
+  }
+  if (HmiInterfaces::STATE_AVAILABLE == tts_info.interface_state &&
+      tts_info.is_unsupported_resource) {
+    tts_response_info_ = "Unsupported phoneme type sent in a prompt";
```

Comment:
@pvvasilev these requirements are applicable for GENIVI & Ford-Specific (specified in Commitment field of ticket). In cited APPLINK-31653:
> SDL must:
> ...
> respond with "WARNINGS, success:true" + info: <message_from_HMI> to mobile app IN CASE 
> HMI respond with UNSUPPORTED_RESOURCE

I think it is exactly this case. 
And requirement regarding info parameter: SDLOPEN-1309 req 4 and 5.